doc/install/get-packages.rst

```diff
@@ -83,7 +83,7 @@ You may find releases for CentOS/RHEL and others (installed with YUM) at::
 
 	https://download.ceph.com/rpm-{release-name}
 
-The major releases of Ceph are summarized at: :doc:`/releases`.
+The major releases of Ceph are summarized at: :doc:`/releases/index`.
```

Comment:
it would be great if you could use a `ref` here too

Reply:
ack